src/integration/pages/mostReadPage/canonicalTests.js

```diff
@@ -0,0 +1,9 @@
+import {
+  runCoreCanonicalTests,
+  // runCanonicalAnalyticsTests,
```

Comment:
Howcome these lines are commented out?

Reply:
This test checks for the ATI related content exists but I've configured that in another PR.

Reply:
PR linked above has been merged now, I've uncommented this test now.